Makefile

```diff
@@ -201,7 +201,11 @@ UI_ROOT        := $(PKG_ROOT)/ui
 SQLPARSER_ROOT := $(PKG_ROOT)/sql/parser
 
 # Ensure we have an unambiguous GOPATH.
-GOPATH := $(shell $(GO) env GOPATH)
+GOPATH := $(realpath $(shell $(GO) env GOPATH))
+# GOPATH need not be explicitly set if it is the default $HOME/go
+ifeq ($(strip $(GOPATH)),)
+GOPATH := $(realpath $(shell echo $$HOME/go))
+endif
```

Comment:
I'd prefer not to commit this workaround yet. I think it's papering over something more serious.